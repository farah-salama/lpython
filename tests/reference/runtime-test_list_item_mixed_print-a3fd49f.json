{
    "basename": "runtime-test_list_item_mixed_print-a3fd49f",
    "cmd": "lpython {infile}",
    "infile": "tests/../integration_tests/test_list_item_mixed_print.py",
    "infile_hash": "14ce4950ca0ff6c6f610df787ad8d260148866f4c7062ab0b856ec5a",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "runtime-test_list_item_mixed_print-a3fd49f.stdout",
    "stdout_hash": "cfcb7e1fe695d586d1eee74aac8a389dacaf75ca596a52f3330ed405",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}