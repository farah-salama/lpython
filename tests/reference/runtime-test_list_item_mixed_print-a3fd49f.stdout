
This is
...
The first element is:
The middle element is:
* 2 =
Total: 9.41999999999999993e+00
(1, 2, 3) is a tuple, but is a number.
1 is smaller than 2 is smaller than 3
1 # 2 # 3 # 4 # 5 # 

List  0 :
List  1 :
List  2 :
